frontend/javascripts/oxalis/controller/combinations/segmentation_handlers.js

```diff
@@ -85,21 +84,3 @@ export async function loadAgglomerateSkeletonAtPosition(position: Vector3) {
 
   await progressCallback(true, "Skeleton generation done.");
 }
-
-export function isosurfaceLeftClick(pos: Point2, plane: OrthoView, event: MouseEvent) {
```

Comment:
`isosurfaceLeftClick` wasn't moved, but was completely removed since it wasn't used anymore. I think, the code become dead due to the new mesh UI we added some time ago.